db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -267,6 +271,15 @@ def up
     end
   rescue PG::ConnectionBad
     say("awx database is unreachable, credentials cannot be migrated")
+  rescue Fernet256::InvalidToken
+    say("awx password invalid, credentials cannot be migrated")
+    say("This is often the case when migrating a support database")
+    say("And fix_auth has reset the ansible secret key")
+    say("")
+    say("If you want to continue and set all the invalid passwords")
+    say("to a preknown bogus value then")
```

Comment:
The word bogus is strange....not sure of an alternative.  Perhaps "a preknown replacement password" ?

Reply:
@Fryguy 'dummy'?

Reply:
I went with "dummy".  But I can change it to what you suggested if you prefer.

Reply:
dummy is perfect